frontend/src/components/Video.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <video ref="video" autoplay="autoplay" />
+</template>
+
+<script>
+  export default {
+    name: 'Video',
+    props: {
+      stream: {
+        type: Object
+      }
+    },
+    mounted() {
+      this.$refs.video.addEventListener('loadeddata', () => {
+        this.$emit('loadeddata')
+      })
```

Comment:
Nice! Love this way of doing it, definitely an improvement from what I had before. :+1: